src/main/java/com/ss/ucm/ms/loans/controllers/LoansController.java

```diff
@@ -0,0 +1,95 @@
+package com.ss.ucm.ms.loans.controllers;
+
+import java.util.Collection;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.ss.ucm.ms.loans.dto.RequestLoanSignupDto;
+import com.ss.ucm.ms.loans.entities.Loan;
+import com.ss.ucm.ms.loans.entities.UserLoan;
+import com.ss.ucm.ms.loans.services.LoanAdd;
+import com.ss.ucm.ms.loans.services.LoanSearch;
+import com.ss.ucm.ms.loans.services.UserLoanAdd;
+
+
+/**
+ * Controller Class for Handling API calls
+ * 
+ * 
+ * @author Charvin Patel
+ */
+
+
+@RestController
+@RequestMapping("/loans")
+@CrossOrigin
+
+public class LoansController {
+    private final LoanSearch loanSearch;
+    
+    @Autowired
+    LoanAdd loanAdd;
+    
+    @Autowired
+    UserLoanAdd userLoanAdd;
+    
+    
+    @Autowired
+    public LoansController(LoanSearch loanSearch) {
+        this.loanSearch = loanSearch;
+    }
+
+    /**
+     * GET /api/loans - Return all loans
+     * @return First 50 loans
+     */
+    @GetMapping
+    public ResponseEntity<Collection<Loan>> get() {
+        try {
+            Collection<Loan> loans = loanSearch.getFirst50();
+            return new ResponseEntity<>(loans, HttpStatus.OK);
+        } catch (Exception e) {
+            return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
+        }
+    }
+    
+    
+	/**
+	 * 
+	 * @param userId, loanRequest(body)
+	 * @return void
+	 * 
+	 */
+    
+    
+	@PostMapping("/loansignup")
```

Comment:
Non-API compliant URI.

Reply:
not sure what API compliant are we even using? Did we discuss about this?

Reply:
It was about Akbar's lecture a few weeks back

More specifically, the URL shouldn't contain verbs and should point to a resource depending on how to reach it. For example, POST .../loan/ would be for posting a new loan. Can just change it later I suppose.